gdpr/anonymizers/model_anonymizers.py

```diff
@@ -196,7 +221,13 @@ def _perform_update(self, obj: Model, updated_data: dict, legal_reason: Optional
                         anonymization: bool = True):
         for field_name, value in updated_data.items():
             setattr(obj, field_name, value)
-        obj.save()
+
+        if is_auditlog_installed():
+            from auditlog.context import disable_auditlog
+            with disable_auditlog():
+                obj.save()
```

Comment:
Does this prevent the creation of another `LogEntry` or what is the purpose here?

Reply:
added comment